airbyte-integrations/connectors/source-jira/metadata.yaml

```diff
@@ -24,6 +24,14 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: "Stream State will be saved for every board in stream `Boards Issues`. Customers who use stream `Board Issues` in Incremental sync mode must take action with their connections."
```

Comment:
Stream state will be saved for every board in stream `Boards Issues`. Customers who use stream `Board Issues` in Incremental Sync mode must take action with their connections.

Reply:
updated